Foundation/NSXMLDTDNode.swift

```diff
@@ -62,7 +62,7 @@ public class NSXMLDTDNode : NSXMLNode {
         }
     } //primitive
     
-    public override init(kind: NSXMLNodeKind, options: Int) {
+    public override init(kind: NSXMLNodeKind, options: NSXMLNodeOptionRawType) {
```

Comment:
I went ahead and took this stuff out.  It's probably not appropriate to mix these into the same PR.  I'll make another one for the XML issues.